app/views/YourSelfAssessmentCharges.scala.html

```diff
@@ -516,9 +506,16 @@ <h2 class="govuk-tabs__title">
                                         }
                                     </div>
                                 }
-                                <div class="govuk-tabs__panel" id="placeholder">
-                                    <h2 class="govuk-heading-l">Placeholder</h2>
-                                </div>
+                            <div class="govuk-tabs__panel govuk-tabs__panel--hidden" id="charges-due-later">
+                                <table class="govuk-table">
+                                    @tableHead("charges-due-later-table-head", showInterest = false)
+                                    <tbody class="govuk-table__body">
+                                    @for((charge, index) <- ChargeItem.chargesDueAfter30DaysList(viewModel.whatYouOweChargesList).zipWithIndex) {
```

Comment:
same here, if we can reduce complexity here and add new property to a view model